molecule/Dockerfile-debian-java8.j2

```diff
@@ -0,0 +1,43 @@
+FROM {{ item.image }}
+
+RUN echo 'deb [check-valid-until=no] http://ftp.debian.org/debian stretch-backports main' | sudo tee /etc/apt/sources.list.d/stretch-backports.list
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends --yes \
+      apt-transport-https \
+      gnupg \
+      software-properties-common \
+      openjdk-8-jdk \
+      rsync \
+      ca-certificates \
+      openssl \
+      unzip \
+      curl
+
+# Hack to fix cacerts on debian hosts, preventing confluenthub installs from working
```

Comment:
Can you explain why this is required?  This is something new that we haven't historically needed.

Reply:
This is fix for ANSIENG-930 and ANSIENG-924